test/unit/sites.js

```diff
@@ -3,179 +3,325 @@
+    site_name: 'fakeSiteName',
+    site_address: '123 Boston, MA'
+};
+const site2 = {
+  site_id: 2,
+  site_name: 'fakeSiteName',
+  site_address: '123 Boston, MA'
+};
+const site3 = {
+  site_id: 3,
+  site_name: 'for the coach',
+  site_address: '123 Boston, MA'
+};
+const site4 = {
+  site_id: 4,
+  site_name: 'also for the coach',
+  site_address: '123 Boston, MA'
+};
+const site5 = {
+  site_id: 5,
+  site_name: 'fakeSiteName',
+  site_address: '123 Boston, MA'
+};
+const site6 = {
+  site_id: 6,
+  site_name: 'fakeSiteName',
+  site_address: '123 Boston, MA'
+};
+const site7 = {
+  site_id: 7,
+  site_name: 'fakeSiteName',
+  site_address: '123 Boston, MA'
+};
+const site8 = {
+  site_id: 8,
+  site_name: 'singe',
+  site_address: 'single Boston, MA'
+};
+const site9 = {
+  site_id: 9,
+  site_name: 'old name',
+  site_address: 'old address'
+};
+const site10 = {
+  site_id: 10,
+  site_name: 'site a',
+  site_address: '370 Marsh Rd, MA'
+};
+const site11 = {
+  site_id: 11,
+  site_name: 'site b',
+  site_address: '144 Atwater St, CA'
+};
+
+describe('Sites', function() {
+  beforeEach(function() {
+    return utils.seed();
+  });
+
+  describe('getSites(req)', function() {
+    it('gets all sites', function(done) {
       var promise = sites.getSites({});
 
       promise.then(function(data) {
-        assert.deepEqual([fakeData1, fakeData2], data);
+        assert.deepEqual([site1, site2, site3, site4, site5, site6, site7, site8, site9, site10, site11], data);
         done();
       });
     });
 
-    xit('gets all sites for a given coach', function(done) {
-      var fakeData1 = {
-        site_id: 1,
-        site_name: 'for the coach',
-        site_address: '123 Boston, MA'
-      };
-      var fakeData2 = {
-        site_id: 2,
-        site_name: 'also for the coach',
-        site_address: '123 Boston, MA'
-      };
+    it('gets all sites for a given coach', function(done) {
       var promise = sites.getSites({
-        req: {
-          query: {
-            coach_id: 1
-          }
+        query: {
+          coach_id: 1
         }
       });
 
       promise.then(function(data) {
-        assert.deepEqual([fakeData1, fakeData2], data);
+        assert.deepEqual([site1, site2], data);
         done();
       });
     });
 
-    xit('create a site', function(done) {
-      var fakeData1 = {
-        site_id: 88888,
-        site_name: 'fakeSiteName',
-        site_address: '123 Boston, MA'
-      };
-      var fakeData2 = {
-        site_id: 99999,
-        site_name: 'fakeSiteName',
-        site_address: '123 Boston, MA'
-      };
+    it('gets all sites for a given coach that does not exsit', function(done) {
+      var promise = sites.getSites({
+        query: {
+          coach_id: 100
+        }
+      });
+
+      promise.then(function(data) {
+        assert.deepEqual([], data);
+        done();
+      });
+    });
+  });
+
+  describe('createSite(req)', function() {
+    it('create a site', function(done) {
```

Comment:
add the test mentioned above, for "don't create site if it already exists" **address before merging**